Ginger/GingerCoreNET/RunLib/CLILib/CLIProcessor.cs

```diff
@@ -87,36 +88,48 @@ private void ParseArgs(string[] args)
             }            
         }
 
-        private int HandleDoOptions(DoOptions opts)
+        private async Task<int> HandleDoOptions(DoOptions opts)
         {
-            DoOptionsHanlder.Run(opts);            
-            return 0;
+           return await Task.Run(() =>
+            {
+                DoOptionsHanlder.Run(opts);
+                return 0;
+            });
+          
         }
 
-        private int HandleVersionOptions(VersionOptions opts)
+        private async Task<int> HandleVersionOptions(VersionOptions opts)
         {
-            PrintGingerVersionInfo();
-            return 0;
+           return await Task.Run(() =>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)